composer.json

```diff
@@ -18,9 +18,9 @@
         }
     ],
     "require": {
-        "php": ">=5.3.2",
-        "symfony/framework-bundle": "~2.7",
-        "symfony/form": "~2.7",
+        "php": ">=5.5.0",
```

Comment:
Good point. I set 5.5.0, because I use the ::class shortcut. But 5.5.9 would have been a better choice to be at least conform with Symfony 3.0.